src/zip/zip.cr

```diff
@@ -0,0 +1,36 @@
+require "zlib"
+require "./*"
+
+# The Zip module contains readers and writers of the zip
+# file format, described at [PKWARE's site](https://pkware.cachefly.net/webdocs/APPNOTE/APPNOTE-6.3.3.TXT).
+#
+# ### Reading zip files
+#
+# Two types are provided to read from zip files:
+# * `Zip::File`: can read zip entries from a `File` or from an `IO::Memory`
+# and provides random read access to its entries.
+# * `Zip::Reader`: can only read zip entries sequentially from any `IO.
+#
+# `Zip::File` is the preferred method to read zip files if you have
+# can provide a `File`, because it's a bit more flexible and provides
+# more complete information for zip entries (such as comments).
```

Comment:
`[...] if you have can provide [...]` — there's either too much or too lil' of sth ;)